tools/truffle/migrations/2_deploy_contracts.js

```diff
@@ -2,6 +2,6 @@ var MyToken = artifacts.require("MyToken");
 
 module.exports = function (deployer) {
   // deployment steps
-  deployer.deploy(MyToken, "8000000000000000000000000", { gas: 4294967295 });
+  deployer.deploy(MyToken, "8000000000000000000000000", { gas: 20000000 });
```

Comment:
Perhaps it's a good idea to use hex here to make it meaningful. So, better use 0xFFFFFFFF to indicate this is the max possible u32. Even better is to use some constant rust function that has that value. I know in C++ it's `std::numeric_limits<uint32_t>::max()`. There's probably a similar functionality in rust.

Reply:
In general I agree with this suggestion. But it seems like @albertov19 is changing it so it is no longer the max value.

Reply:
Yes exactly. This is just use as an example. So when people used the gas 4294967295 on TestNet it broke their deployment there so I prefer to put a much safer vale (in decimal so its human readable). We can actually remove the gas altogether now because this was initially that our gas estimations were wrong.

@JoshOrndorff the tools folder is for things such as truffle which is a tool that people can use to deploy contracts. I guess we can remove the truffle folder once the Box is officially out.